src/index.js

```diff
@@ -1 +1,21 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { getScores, addScore } from './modules/scoresHandle.js';
+import { displayScores } from './modules/createList.js';
+
+const submitBtn = document.querySelector('.btn-submit');
+const refreshBtn = document.querySelector('.btn-refresh');
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  addScore();
+  getScores().then((response) => {
+    displayScores(response.result);
+  });
+});
+
+refreshBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScores().then((response) => {
+    displayScores(response.result);
+  });
```

Comment:
- [ ] [OPTIONAL] We are encouraged to use Object Destructuring in ES6. This block of code can also be written as:

```JavaScript
refreshBtn.addEventListener('click', (e) => {
  e.preventDefault();
  getScores().then(({ result }) => {
    displayScores(result);
  });
});
```
Notice this line where the Destructuring has taken place 👉🏼 ```getScores().then(({ result }) => {```

You can consider refactoring this.